fhir-server/pom.xml

```diff
@@ -105,6 +105,11 @@
             <artifactId>fhir-persistence-jdbc</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-persistence-cassandra</artifactId>
```

Comment:
some discussion on whether to include this in the server by default or not... I'm leaning toward including it since it requires explicit action.  but it still adds all the jars to the classpath which increases attack surface area.

Reply:
One option would be make the Cassandra variant experimental, but include the COS/Azure Blob variants (depending on our support)

Reply:
fhir-persistence-cassandra no longer included in fhir-server pom.